test/string_test.rb

```diff
@@ -273,6 +273,29 @@
     end
   end
 
+  describe '#singularize' do
+    before do
+      @singular, @plural = *TEST_SINGULARS.to_a.sample
```

Comment:
not a big fan of testing against a moving random value, can we do it on fixed value instead?